src/main/java/com/github/luben/zstd/BaseZstdBufferDecompressingStreamNoFinalizer.java

```diff
@@ -27,6 +33,9 @@ protected ByteBuffer refill(ByteBuffer toRefill) {
         return toRefill;
     }
 
+    /**
+     * @return true if all decompressed data from the has been read and no more data is available in {@link #source}
```

Comment:
it's the other way around - @return false if all data is processed and no more data is available from the source

Reply:
Ah good catch! Fixed it in latest commit.